org.gtk.Gtk3theme.High-Sierra.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+    <id>org.gtk.Gtk3theme.High-Sierra</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <name>High Sierra GTK Theme</name>
+    <summary>High Sierra like theme for GTK 2 and GTK 3.</summary>
```

Comment:
Might as well just remove the GTK 2 reference since this is a GTK3 only package.

Reply:
done remove it...